modules/system/defaults/NSGlobalDomain.nix

```diff
@@ -225,6 +225,24 @@ in {
       '';
     };
 
+    system.defaults.NSGlobalDomain.NSStatusItemSpacing = mkOption {
+      type = types.nullOr types.int;
+      default = null;
+      example = 12;
+      description = lib.mdDoc ''
+        Sets the spacing between status icons in the menu bar. The default is null.
```

Comment:
The description should specify the macOS default like the rest of the options

https://github.com/LnL7/nix-darwin/blob/master/modules/system/defaults/NSGlobalDomain.nix#L11-L17

Reply:
Those are the MacOS defaults. Both of these options are unset by default.